core/src/main/kotlin/com/github/inflab/spring/data/mongodb/core/aggregation/expression/comparison/CmpExpressionDsl.kt

```diff
@@ -0,0 +1,104 @@
+package com.github.inflab.spring.data.mongodb.core.aggregation.expression.comparison
+
+import com.github.inflab.spring.data.mongodb.core.aggregation.expression.AggregationExpressionDsl
+import com.github.inflab.spring.data.mongodb.core.annotation.AggregationMarker
+import com.github.inflab.spring.data.mongodb.core.extension.toDotPath
+import org.bson.Document
+import org.springframework.data.mongodb.core.aggregation.AggregationExpression
+import kotlin.reflect.KProperty
+
+/**
+ * A Kotlin DSL to configure `$cmp` [CmpExpressionDsl] using idiomatic Kotlin code.
+ *
+ * @author minwoo
+ * @since 1.0
+ * @see <a href="https://docs.mongodb.com/manual/reference/operator/aggregation/cmp">$cmp</a>
+ */
+@AggregationMarker
+class CmpExpressionDsl {
+    /**
+     * Represents all operands inside `$cmp` as a list.
+     *
+     * @property values The list that contains all operands.
+     */
+    @JvmInline
+    value class Operands(val values: MutableList<Any>)
```

Comment:
we can use `Cmp` expression from `Spring Data Mongodb`. 
then we can remove `build` method.